src/style.css

```diff
@@ -1 +1,84 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 400;
+}
+
+html {
+  font-weight: 300;
+}
+
+.search-bar__input {
+  width: 100%;
+  display: flex;
```

Comment:
Using 'display: flex;' on an input element is unusual and may not have the intended effect. Consider if this is necessary or if it should be applied to a parent container instead.